deblur/deblurring.py

```diff
@@ -55,7 +55,34 @@ def get_sequences(input_seqs):
     return seqs
 
 
-def deblur(input_seqs, read_error=0.05, mean_error=0.005, error_dist=None,
+def create_error_profile(read_error):
+    """Create a theoretical read error profile given
+    a maxima error rate
+
+    Parameters
+    ----------
+    read_error : float
+        the upper bound on the hamming one read error
+
+    Results
+    -------
+    error_profile : list of float
+        the predicted theoretical error profile
+        (note for hamming>1 we take the max of
+        read_error^2 and 0.01 since closer to reality)
+    """
+    error_profile = [1.0, read_error, np.max([0.01, np.power(read_error, 2)]),
+                     np.max([0.01, np.power(read_error, 3)]),
+                     0.01, 0.005, 0.005, 0.005, 0.005, 0.005, 0.005, 0.001,
+                     0.001, 0.001, 0.001]
+    return error_profile
+
+
+def deblur(input_seqs, mean_error=0.005,
+           error_dist=[1, 0.05, 0.000005, 0.000005, 0.000005, 0.000005,
```

Comment:
We want to be able to write the default error profile in the deblur workflow --help. If we would just use the defaults in this function, we won't know what to print...
Maybe best to have a function that returns the default error profile, and have deblur() just call it if error_dist is None?
Anyway, this default inside deblur() is not supposed to be used when running via the command line